webui/react/src/components/Link.tsx

```diff
@@ -6,17 +8,22 @@ import css from './Link.module.scss';
 
 interface Props {
   disabled?: boolean;
+  inherit?: boolean;
   path: string;
   popout?: boolean;
   onClick?: (event: React.MouseEvent) => void;
 }
 
+const windowFeatures = [ 'noopener', 'noreferrer' ];
```

Comment:
> Note that when noopener is used, nonempty target names other than _top, _self, and _parent are all treated like _blank in terms of deciding whether to open a new window/tab.

this wouldn't affect us right?

Reply:
Yeah we are ok for now. Right we only link internally to links that we control, but if as soon as we introduce a link that's outside a domain that we control, this will be good to have.